src/main/java/htsjdk/samtools/util/BufferedLineReader.java

```diff
@@ -80,7 +79,7 @@ public String readLine() {
     @Override
     public int peek() {
         try {
-            mark(1);
```

Comment:
Could you add a comment here explaining this, it will be confusing for future generations otherwise.

Reply:
Good idea.